Source/SpacePort/SpacePort/Models/Receipt.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace SpacePort.Models
+{
+    public class Receipt
```

Comment:
You can use the attribute `[Key]` to denote that the "ID"-property uniquely identifies this entity, this maps the property as a "Primary Key" in the database.

Reply:
Ye, we had that, but removed it when we trying out to fix one-to-one error, just to eleiminate an potential error-source :) , forgot to put it back